testdata/script/literals.txtar

```diff
@@ -25,9 +25,15 @@ binsubstr main$exe 'Lorem Ipsum' 'dolor sit amet' 'second assign' 'First Line' '
 # seconds, it means we're trying to obfuscate them.
 generate-literals extra_literals.go
 
-garble -literals -debugdir=debug1 build
+# ensure we find the extra literals in an unobfuscated build
+go build -o=out
```

Comment:
avoid using `-o=out`; use the default `main$exe` like the rest of the test for consistency.

Reply:
ups will do. now I understand what this strange `main$exe` is... :D